plugins/plugin.proto

```diff
@@ -56,22 +56,33 @@ message Version {
   string suffix = 4;
 }
 
-// An encoded PluginRequest is written to the plugin's stdin.
-message PluginRequest {
+// A parameter passed to the plugin from (or through) the OpenAPI compiler.
+message Parameter {
+  // The name of the parameter as specified in the option string
+  string name = 1;
+
+  // The parameter value as specified in the option string
+  string value = 2;
+}
 
-  // The OpenAPI descriptions that were explicitly listed on the command line.  
-  // The specifications will appear in the order they are specified to openapic.
-  repeated Wrapper wrapper = 1;
+// An encoded Request is written to the plugin's stdin.
+message Request {
+
+  // A wrapped OpenAPI document to process.
+  Wrapper wrapper = 1;
   
-  // The plugin parameter passed on the command-line.
-  string parameter = 2;
+  // The plugin invocation string, usually specified as a command-line option to the OpenAPI compiler.
```

Comment:
OK, it's probably not needed now since we're strictly enforcing the parameter syntax. I'll remove it.